php/example_code/phpunit.xml

```diff
@@ -8,10 +8,13 @@
          failOnRisky="true"
          failOnWarning="true"
          verbose="true">
-    <testsuites>
-        <testsuite name="kms-basics">
-            <file>kms/tests/KmsBasicsTests.php</file>
-        </testsuite>
+<!--    <testsuites>-->
+<!--        <testsuite name="kms-basics">-->
+<!--            <file>kms/tests/KmsBasicsTests.php</file>-->
+<!--        </testsuite>-->
+<!--        <testsuite name="s3-express-basics">-->
+<!--            <file>s3/tests/S3ExpressBasicsTest.php</file>-->
+<!--        </testsuite>-->
```

Comment:
Delete?

Reply:
I use this for localized testing of individual services, so I keep the commented out sections for newly added services so it's easy to turn them on and off.